scripts/clean-local-repos.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+repos_dir="${HOME}/repos"
+exclude_dir=("$@")
+
+folders=$(find "${repos_dir}" -maxdepth 1 -mindepth 1 -type d -exec basename {} \;)
+if [[ ${#exclude_dir[@]} -ne 0 ]]; then
+  folders=$(echo "${folders}" | grep -v -E "$(IFS="|"; echo "${exclude_dir[*]}")")
+fi
```

Comment:
we can simplify this, just add the exclusions in find command itself. Raw example:

```bash
cd $repos_dir
repos=$(find . -type d ! -name "felipe" ! -name "cassio")
```